guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+
+#### Federation Metadata Fields: Explanation and Usage
+
+Federations can provide configuration and metadata to users. These metadata fields are consensus-relevant, meaning they must be consistent across all federation members. This ensures that users can rely on their accuracy. Let's explore the core metadata fields defined in the Fedimint protocol:
+
+#### Federation Overview and Expiration
+
+<div class="center" markdown="1">
+
+{% include image-gallery.html pages = page.images_guardian-expiry %}
+
+- `federation_name`: This field provides the name of the federation. It should be displayed prominently in the federation overview to help users quickly identify which federation they are interacting with.
+
+- `federation_expiry_timestamp`: This field contains a timestamp indicating when the federation will shut down. It's crucial to display this information clearly to users, possibly with a countdown or warning as the expiration date approaches.
+
+</div>
+
+#### User Onboarding and Information
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/federation-welcome.png"
+   retina = "/assets/images/guide/how-it-works/ecash/federation-welcome@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/ecash/federation-welcome@2x.png"
+   alt-text = "Federation welcome message display in a wallet interface."
+   width = 250
+   height = 321
+   layout = "float-right-desktop -background -shadow"
+   caption = "A welcome message being used durring the new user onboarding process."
+%}
+
+- `welcome_message`: This field contains a welcome message for new users joining the federation. It could be displayed during the onboarding process to provide users with important information or instructions.
+
+</div>
+
+{% include tip/open.html color="blue" icon="info" label="Flexible Use of Welcome Message" %}
+Different federations may choose to display the welcome message in different contexts. Some may use it as a welcome message, others may use it to display the terms of service or other important information.
+{% include tip/close.html %}
+
+#### Gateway Information
+
+- `vetted_gateways`: This field contains a list of gateway identifiers that have been vetted by the federation. This information could be used to display trusted gateways to users, helping them make informed decisions about which gateways to use for transactions.
+
+##### What Is a Gateway?
+See the [Role of Gateways]({{ '/guide/how-it-works/ecash/fedimint/#role-of-gateways' | relative_url }}) for more information about gateways.
+
+#### Why Showing Gateway Information Might Not Be Necessary:
+
+1. **Cognitive Overload**: Most users are not interested in the technical details of how their transactions are processed. Presenting information about gateways, especially vetted versus non-vetted ones, could lead to confusion, as users may not understand what this information means or how to act on it.
```

Comment:
How about showing something about vetted gateways at the point they become relevant to users? I do think people care, but it doesn't have to be front-loaded or too in-your-face.

Reply:
I'll have to find out at what point they become relevant to users. I can't think of one. @elsirion any thoughts?